.github/workflows/docs.yml

```diff
@@ -4,11 +4,13 @@ on:
   push:
     branches:
       - dev
+  pull_request:
+    branches:
+      - dev
 
 jobs:
   build-docs:
     runs-on: ubuntu-latest
-    name: Build documentation
```

Comment:
two same names are kind of duplicated
![image](https://user-images.githubusercontent.com/25895405/198552199-672b2c9c-08bb-4ce0-969d-6686adf7845b.png)